CMakeLists.txt

```diff
@@ -23,6 +23,8 @@ if (NOT EMSCRIPTEN AND NOT IOS)
     set(ALSOFT_EXAMPLES OFF CACHE BOOL "")
     set(ALSOFT_TESTS OFF CACHE BOOL "")
     set(CMAKE_DISABLE_FIND_PACKAGE_WindowsSDK ON CACHE BOOL "")
+    # Ensure the redistributables are compiled in for openal
+    set(FORCE_STATIC_VCRT ON)
```

Comment:
Can the option be set after the sub dir include or at least after the message prompt below?

Reply:
We may consider wrapping it in a condition for Windows/MSVC only

Reply:
I think this also needs to check BUILD_SHARED_LIBS too as we link the dynamic runtime when building SFML dynamic libs?

Reply:
Made the requested changes, however the set must be before the add_subdirectory.

@JonnyPtn these are separate, and pre-built, so idk. For the past 7 years, x86 and x64 used the build with the vc_redistributables included, so I have no idea...